plugins/hls-eval-plugin/src/Ide/Plugin/Eval/Code.hs

```diff
@@ -81,15 +80,17 @@ asStmts (Example e _ _) = NE.toList e
 asStmts (Property t _ _) =
     ["prop11 = " ++ t, "(propEvaluation prop11 :: IO String)"]
 
--- |GHC extensions required for expression evaluation
-evalExtensions :: [Extension]
-evalExtensions =
-    [ OverlappingInstances
-    , UndecidableInstances
-    , FlexibleInstances
-    , IncoherentInstances
-    , TupleSections
-    ]
+-- -- |GHC extensions required for expression evaluation
+-- evalExtensions :: [Extension]
+-- evalExtensions =
+--     [ NondecreasingIndentation
+--     -- , OverlappingInstances
+--     -- , UndecidableInstances
+--     -- , FlexibleInstances
+--     -- , IncoherentInstances
+--     -- , TupleSections
+--     , ExtendedDefaultRules
+--     ]
```

Comment:
```suggestion
```

Reply:
Hi @berberman 
Sorry, I forgot to delete the comment. Thanks for spotting this!
FM